knora/excel2xml.py

```diff
@@ -396,6 +398,11 @@ def make_resource(
     if ark and iri:
         warnings.warn(f"Both ARK and IRI were provided for resource '{label}' ({id}). The ARK will override the IRI.",
                       stacklevel=2)
+    if creation_date:
+        validate_resource_creation_date(creation_date,
+                                        f"The resource '{label}' (ID: {id}) has an invalid creation date. Did you "
+                                        f"perhaps forget the timezone?")
```

Comment:
very much a matter of taste, but I much prefer the following indentation schema over what PyCharm does. (The indentation also lead to more commonly needing to split string arguments.)

```suggestion
        validate_resource_creation_date(
            creation_date,
            f"The resource '{label}' (ID: {id}) has an invalid creation date. Did you perhaps forget the timezone?"
        )
```